README.md

```diff
@@ -1,3 +1,7 @@
 # Rainy Days Course Assignment
 
 This is a repo for an HTML and CSS course assignment
+
+### Execute program
+
+[https://splendid-custard-e9a90d.netlify.app](https://splendid-custard-e9a90d.netlify.app)
```

Comment:
Well done on adding the link

Reply:
You can as well add the link to your prototype

Reply:
Sorry. This is the prototype:
https://www.figma.com/file/3c0eAlUutWaUNcB2Dt0Rzp/Rainy-Days?node-id=0%3A1